Clear up common questions releases

zzak opened the pull request:
We often get asked about bug fixes for unsupported series, and NPM security release version scheme.

cc #47390, #39351, #47216

eileencodes commented on guides/source/maintenance_policy.md:
```diff
@@ -49,6 +49,10 @@ theoretical 1.2.2 patch release would be built from the 1-2-stable branch.
 
+For unsupported series, bug fixes may coincidentally land in a stable branch,
+but are unreleased in an official version. It is recommended to point your
+application at the stable branch using git.
```
```suggestion
application at the stable branch using Git for unsupported versions.
```

eileencodes commented on guides/source/maintenance_policy.md:
```diff
@@ -49,6 +49,10 @@ theoretical 1.2.2 patch release would be built from the 1-2-stable branch.
 they are included in the list of supported series.
 
+For unsupported series, bug fixes may coincidentally land in a stable branch,
+but are unreleased in an official version. It is recommended to point your
```
```suggestion
but won't be released in an official version. It is recommended to point your
```

eileencodes commented on guides/source/maintenance_policy.md:
```diff
@@ -85,6 +94,14 @@ Unsupported Release Series
-publish them to git, however there will be no new versions released. If you are
-not comfortable maintaining your own versions, you should upgrade to a
-supported version.
+publish them to git, however there will be no new versions released. We
```
```suggestion
merge them, however there will be no new versions released. We
```

eileencodes commented on guides/source/maintenance_policy.md:
```diff
@@ -85,6 +94,14 @@ Unsupported Release Series
-not comfortable maintaining your own versions, you should upgrade to a
-supported version.
+publish them to git, however there will be no new versions released. We
+recommend to point your application at the stable branch using git. If you are
```
```suggestion
recommend to point your application at the stable branch using Git. If you are
```

eileencodes reviewed (commented):
Thank you, I feel like a broken record lately 😆

zzak committed:
Clear up common questions releases

We often get asked about bug fixes for unsupported series, and NPM security release version scheme.

cc #47390, #39351, #47216

Co-authored-by: Eileen M. Uchitelle <eileencodes@users.noreply.github.com>
